.quickstart/quickstart.yml

```diff
@@ -0,0 +1,31 @@
+dbt_versions: '>=1.3.0 <2.0.0'
+
+multi_connector_config:
+  facebook_pages:
+    database_key: facebook_pages_database
+    union_schema_key: facebook_pages_union_schemas
+    model_variable: social_media_rollup__facebook_enabled
+  instagram_business:
+    database_key: instagram_business_database
+    union_schema_key: instagram_business_union_schemas
+    model_variable: social_media_rollup__instagram_enabled
+  linkedin_company_pages:
+    database_key: linkedin_pages_database
+    union_schema_key: linkedin_pages_union_schemas
+    model_variable: social_media_rollup__linkedin_enabled
+  twitter:
+    database_key: twitter_organic_database
+    union_schema_key: twitter_organic_union_schemas
+    model_variable: social_media_rollup__twitter_enabled
+
+table_variables: {}
```

Comment:
@fivetran-joemarkiewicz do we need this line? i noticed in the quickstart.yml files for the individual packages, we just don't include the table_variables section

Reply:
@fivetran-jamie I would lean on the side of keeping it. @FedorSafronov added this for configuration of combo data models and if that was included I would rather keep it than remove it to make sure it doesn't cause unexpected issues.